src/components/dialogs/csv-import-filter-creation-dialog.js

```diff
@@ -131,7 +131,6 @@ const CsvImportFilterCreationDialog = ({
                 <Dialog
                     open={isConfirmationPopupOpen}
                     aria-labelledby="dialog-confirmation-csv-data"
-                    onKeyPress={() => handleAddPopupConfirmation()}
```

Comment:
Maybe delete the associated function ?

```
    const handleAddPopupConfirmation = () => {
        handleCreateFilter(true);
        setOpenConfirmationPopup(false);
    };
```

Reply:
No, it is used on line 153 : 
`onClick={() => handleAddPopupConfirmation()}`

Reply:
Ah yes sorry